src/App.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <v-app>
-    <Appbar @toggle-sidebar="toggleSidebar" />
+    <Appbar style="z-index: 999" @toggle-sidebar="toggleSidebar" />
```

Comment:
Don't use inline style—move everything to the `style` tag.

Reply:
Was trying to overwrite some of the Vuetify stylesheets XD
cant find a better way.. for now. I believe I tried to use "!important" but sometimes it doesn't work.
Worried if I change the configuration files directly it might be overwritten when Vuetify updates..